src/tools/server-tools.js

```diff
@@ -28,6 +28,10 @@ export function connectionModeIsServer(state) {
   return getConnectionMode(state) === "SERVER";
 }
 
+export function notebookIsATrial(state) {
+  return state.notebookInfo.tryItMode;
```

Comment:
`tryItMode` should be added to the state schema
https://github.com/iodide-project/iodide/blob/ae55936a5b9250215f41573cda2a69a68212253a/src/state-schemas/state-schema.js#L185